machine-learning-image-classification-with-javascript-and-nyckel/index.md

```diff
@@ -0,0 +1,352 @@
+### Machine Learning Image Classification with Javascript and Nyckel.
+
+### Introduction
+Artificial intelligence has become the future of technology, and everyone is trying to apply it in all areas. Artificially intelligent applications contribute to AI technology, with trading bots and others. Web developers, too, want to integrate these models into their applications.
+
+This article shows how to implement machine learning models in a web application to classify images. In this case, we will classify and identify the images of people wearing and those not wearing a mask.
+
+### Table of Contents
+- [Introduction](#introduction)
+- [Requirements](#requirements)
+- [JavaScript CSS and HTML website build](#javascript-css-and-html-website-build)
+- [Building the Machine Learning model using Nyckel](#building-the-machine-learning-model-using-nyckel)
+- [Machine learning model integration with the Web Application](#machine-learning-model-integration-with-the-web-application)
+- [App live testing](#app-live-testing)
+- [Conclusion](#conclusion)
+
+### Requirements
+To develop, you will need the following;
+- A suitable editor like visual studio, sublime, and others.
+- Prior knowledge of Nyckel, css, JavaScript.
+- HTML basics.
+
+### What is Nyckel?
+Nyckel is a machine learning API that provides automated training to models instead of manually writing the code. It also allows these models to be integrated into applications with the best simplicity.
+
+With these requirements in place, the coding journey can kick-off.
+Follow each step below to develop your own machine learning-enabled web application.
+
+### JavaScript CSS and HTML website build
+We will build a simple web application to upload a photo from the desktop into the app. The layout will only need a section to place your photo and a button to make a system call to the file explorer.
+
+#### **HTML Build-up**
+Below are the steps involved in HTML build-up. First, create a file with a `.html` extension and follow the following steps.
+This code is placed under the `<head>` tag creates a connection to the JavaScript file.
+```html
+<html>
+
+<head>
+  <script src="https://code.jquery.com/jquery-latest.min.js"></script>
+</head>
+
+```
+
+Under the body, the tag is where the upload button and the upload section lies. This code creates what is required under the body section without any styling.
+```html
+<body>
+
+  <div id="main" class="container">
+
+    <h1 id="title">Mask or no mask?</h1>
+
+    <!-- Drag and Drop container-->
+    <div class="upload-area" id="uploadfile">
+      <h2>Drop Your photo here</h2>
+    </div>
+    <input type="file" name="file" id="file">
+    <a id="nyckel" href="https://www.nyckel.com">Machine Learning application created by Victor</a>
+  </div>
+</body>
+```
+
+Once uploaded, a container is created to hold the photo using the division class `upload-area`.

```

Comment:
Once uploaded? what's being uploaded here while you're describing how to build...